src/routers/secure/items.py

```
@@ -432,4 +433,71 @@ async def unblacklist_stream(_: Request, item_id: str, stream_id: int, db: Sessi
+                    all_ids = [item_id] + related_ids
+
+                    for id in all_ids:
+                        request.app.program.em.cancel_job(id)
+                        request.app.program.em.remove_id_from_queues(id)
+
+                    if media_item.last_state not in [States.Paused, States.Failed, States.Completed]:
+                        media_item.store_state(States.Paused)
+                        session.merge(media_item)
+                        session.commit()
+
+                    logger.info(f"Successfully paused items.")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)